__tests__/clock.test.ts

```diff
@@ -0,0 +1,28 @@
+import { DydxClient } from '../src';
+
+describe('Clock', () => {
+  it('constructor', async () => {
+    const client = new DydxClient('https://example.com', {});
+    expect(client.clock.timestampAdjustment).toEqual(0);
+  });
+
+  it('setTimestampAdjustment', async () => {
+    const newAdjustment: number = 12.123;
+    const client = new DydxClient('https://example.com', {});
+    client.clock.setTimestampAdjustment(newAdjustment);
+    expect(client.clock.timestampAdjustment).toEqual(newAdjustment);
+  });
+
+  it('getAdjustedIsoString', async () => {
+    const newAdjustment: number = -10.5;
```

Comment:
nit - I didn't originally know `timestamp.setSeconds/timestamp.getSeconds()` could be used that way, consider testing with something a lot bigger than 60 seconds